Description: Autogenerated stuff changes during build
 Upstream tarball contains autogenerated stuff, which looks slightly different
 being built with our tool chain.
Origin: other

--- python-cloudservers-1.0~a5.orig/python_cloudservers.egg-info/PKG-INFO
+++ python-cloudservers-1.0~a5/python_cloudservers.egg-info/PKG-INFO
@@ -29,54 +29,54 @@ Description: Python bindings to the Rack
         with the ``--username`` and ``--apikey`` params, but it's easier to just set
         them as environment variables::
         
-        export CLOUD_SERVERS_USERNAME=jacobian
-        export CLOUD_SERVERS_API_KEY=yadayada
-        
+            export CLOUD_SERVERS_USERNAME=jacobian
+            export CLOUD_SERVERS_API_KEY=yadayada
+            
         You'll find complete documentation on the shell by running ``cloudservers help``::
+            
+            usage: cloudservers [--username USERNAME] [--apikey APIKEY] <subcommand> ...
         
-        usage: cloudservers [--username USERNAME] [--apikey APIKEY] <subcommand> ...
-        
-        Command-line interface to the Cloud Servers API.
+            Command-line interface to the Cloud Servers API.
         
-        Positional arguments:
-        <subcommand>
-        backup-schedule     Show or edit the backup schedule for a server.
-        backup-schedule-delete
-        Delete the backup schedule for a server.
-        boot                Boot a new server.
-        delete              Immediately shut down and delete a server.
-        flavor-list         Print a list of available 'flavors' (sizes of
-        servers).
-        help                Display help about this program or one of its
-        subcommands.
-        image-create        Create a new image by taking a snapshot of a running
-        server.
-        image-delete        Delete an image.
-        image-list          Print a list of available images to boot from.
-        ip-share            Share an IP address from the given IP group onto a
-        server.
-        ip-unshare          Stop sharing an given address with a server.
-        ipgroup-create      Create a new IP group.
-        ipgroup-delete      Delete an IP group.
-        ipgroup-list        Show IP groups.
-        ipgroup-show        Show details about a particular IP group.
-        list                List active servers.
-        reboot              Reboot a server.
-        rebuild             Shutdown, re-image, and re-boot a server.
-        rename              Rename a server.
-        resize              Resize a server.
-        resize-confirm      Confirm a previous resize.
-        resize-revert       Revert a previous resize (and return to the previous
-        VM).
-        root-password       Change the root password for a server.
-        show                Show details about the given server.
-        
-        Optional arguments:
-        --username USERNAME   Defaults to env[CLOUD_SERVERS_USERNAME].
-        --apikey APIKEY       Defaults to env[CLOUD_SERVERS_API_KEY].
-        
-        See "cloudservers help COMMAND" for help on a specific command.
+            Positional arguments:
+              <subcommand>
+                backup-schedule     Show or edit the backup schedule for a server.
+                backup-schedule-delete
+                                    Delete the backup schedule for a server.
+                boot                Boot a new server.
+                delete              Immediately shut down and delete a server.
+                flavor-list         Print a list of available 'flavors' (sizes of
+                                    servers).
+                help                Display help about this program or one of its
+                                    subcommands.
+                image-create        Create a new image by taking a snapshot of a running
+                                    server.
+                image-delete        Delete an image.
+                image-list          Print a list of available images to boot from.
+                ip-share            Share an IP address from the given IP group onto a
+                                    server.
+                ip-unshare          Stop sharing an given address with a server.
+                ipgroup-create      Create a new IP group.
+                ipgroup-delete      Delete an IP group.
+                ipgroup-list        Show IP groups.
+                ipgroup-show        Show details about a particular IP group.
+                list                List active servers.
+                reboot              Reboot a server.
+                rebuild             Shutdown, re-image, and re-boot a server.
+                rename              Rename a server.
+                resize              Resize a server.
+                resize-confirm      Confirm a previous resize.
+                resize-revert       Revert a previous resize (and return to the previous
+                                    VM).
+                root-password       Change the root password for a server.
+                show                Show details about the given server.
+        
+            Optional arguments:
+              --username USERNAME   Defaults to env[CLOUD_SERVERS_USERNAME].
+              --apikey APIKEY       Defaults to env[CLOUD_SERVERS_API_KEY].
         
+            See "cloudservers help COMMAND" for help on a specific command.
+            
         Python API
         ----------
         
@@ -86,32 +86,32 @@ Description: Python bindings to the Rack
         
         By way of a quick-start::
         
-        >>> import cloudservers
-        >>> cs = cloudservers.CloudServers(USERNAME, API_KEY)
-        >>> cs.flavors.list()
-        [...]
-        >>> cs.servers.list()
-        [...]
-        >>> s = cs.servers.create(image=2, flavor=1, name='myserver')
-        
-        ... time passes ...
-        
-        >>> s.reboot()
-        
-        ... time passes ...
-        
-        >>> s.delete()
+            >>> import cloudservers
+            >>> cs = cloudservers.CloudServers(USERNAME, API_KEY)
+            >>> cs.flavors.list()
+            [...]
+            >>> cs.servers.list()
+            [...]
+            >>> s = cs.servers.create(image=2, flavor=1, name='myserver')
+            
+            ... time passes ...
+            
+            >>> s.reboot()
+            
+            ... time passes ...
+            
+            >>> s.delete()
         
         FAQ
         ===
         
         What's wrong with libcloud?
         
-        Nothing! However, as a cross-service binding it's by definition lowest
-        common denominator; I needed access to the Rackspace-specific APIs (shared
-        IP groups, image snapshots, resizing, etc.). I also wanted a command-line
-        utility.
-        
+            Nothing! However, as a cross-service binding it's by definition lowest
+            common denominator; I needed access to the Rackspace-specific APIs (shared
+            IP groups, image snapshots, resizing, etc.). I also wanted a command-line
+            utility.
+            
         
 Platform: UNKNOWN
 Classifier: Development Status :: 3 - Alpha
